mgmtsystem/azure.py

```diff
@@ -297,6 +297,7 @@ def delete_stack(self, stack_name, resource_group=None):
             -DeploymentName \"{stack}\" -Force
             }}
             """.format(rg=resource_group or self.resource_group, stack=stack_name))
+        self.delete_vm(stack_name, resource_group or self.resource_group)
```

Comment:
@jteehan , @psav  is right here, we are not the only ones leveraging wrapanapi now, the Katello team is also using this for their provider integrations.  We need to devise a better method of tracking with vms in the stack before the stack deletion which we can then process after the fact to make sure they were cleaned up.